coralproject/talk
# Autoindexing scanned comment fields

## What does this PR do?

Adds indexes to commonly queried fields: comment.asset_id and comment.parent_id.

## How do I test this PR?

Start a fresh database. Check to see that the indexes are added on server start.
